### PR TITLE
Feature/123 set husky hooks

### PR DESCRIPTION
**Pull request by rodrigo-baldiviezo-jalafund**
[AB#123](https://dev.azure.com/funjaladev/DevLevel3/_workitems/edit/123/)

Set husky hooks

- pre-commit: fix code with prettier
- pre-push: run linting and testing
- post-checkout: install the dependencies (npm install or yarn)

### CODE REVIEW

File: .husky/pre-commit
Comment (fernando-maceda-jalafund) [inline .husky/pre-commit:4]: You should probably need to have npx as a dev dependency

File: package.json
Comment (fernando-maceda-jalafund) [inline package.json:22]: it seems you have two scripts doing the same "prettier --write"

File: .husky/post-checkout
Comment (pedro-lima-jalafund) [inline .husky/post-checkout]: maybe you can change this to `npm install` in order to be more explicit.

**Review by pedro-lima-jalafund — CHANGES_REQUESTED** [review]

**Review by pedro-lima-jalafund — DISMISSED** [review]

**Review by fernando-maceda-jalafund — COMMENTED** [review]
Take a look please

File: package.json
Comment (rodrigo-baldiviezo-jalafund) [reply package.json:22]: Yes a test just with one, but when run, format all the files. It's because --write .
The dot cause the problem. In lint-staged object is selected with * all files but in the staged

**Review by ela-ocana-jalafund — APPROVED** [review]

**Review by fernando-maceda-jalafund — APPROVED** [review]
LGTM. I will merge the PR once it has the necessary approvals

**Review by pedro-lima-jalafund — APPROVED** [review]

**Pull request merged**
